### PR TITLE
CSP related changes

### PR DESCRIPTION
**Pull request by mtrojan-ub**

### CODE REVIEW

File: themes/tuefind/js/tuefind.js [hunk 1]
```diff
@@ -309,6 +310,15 @@ $(document).ready(function () {
     } else if (window.location.pathname.match(/\/Alphabrowse/i)) {
         TueFind.SetFocus('#alphaBrowseForm_from');
     }
+    
+    $(".tuefind-event-resetsearchhandlers").click(function(){
+        TueFind.ResetSearchHandlers(); 
+        return TueFind.CheckWildcards(event);
+    })
```
Comment (mtrojan-ub) [inline themes/tuefind/js/tuefind.js:317]: missing semicolon at end of line (1)

File: themes/tuefind/js/tuefind.js [hunk 2]
```diff
@@ -309,6 +310,15 @@ $(document).ready(function () {
...
+    
+    $('.tuefind-event-searchForm-on-submit').submit(function(){
+        return TueFind.CheckWildcards(event);
+    })
```
Comment (mtrojan-ub) [inline themes/tuefind/js/tuefind.js:321]: missing semicolon at end of line (2)

**Comment by mtrojan-ub** [comment]
Please also change the following file, as discussed:
tuefind/templates/RecordTab/itemfulltextsearch.phtml

File: themes/tuefind/templates/RecordTab/itemfulltextsearch.phtml [hunk 3]
```diff
@@ -34,14 +34,20 @@
...
+<span>
+    <input id="homeUrl" type="hidden" value="<?=$this->url('home')?>"/>
+    <input id="uniqueID" type="hidden" value="<?=$this->driver->getUniqueID()?>"/>
+    <input id="fulltextTypes" type="hidden" value="<?="'" . implode(',', $this->driver->getFulltextTypes()) ."'"?>"/>
+</span>
+
 <form class="searchForm form-horizontal" name="ItemFulltextSearchForm" id="ItemFulltextSearchForm" autocomplete="off"
-  onSubmit="javascript:TueFind.GetFulltextSnippets(&quot;<?=$this->url('home')?>&quot;,
-                                                   &quot;<?=$this->driver->getUniqueID()?>&quot;,
+  onSubmit="javascript:TueFind.GetFulltextSnippets(document.getElementById('homeUrl').value,
```
Comment (mtrojan-ub) [inline themes/tuefind/templates/RecordTab/itemfulltextsearch.phtml:44]: the whole onsubmit block should be moved to tuefind.js too

File: themes/tuefind/templates/RecordTab/itemfulltextsearch.phtml [hunk 4]
```diff
@@ -34,14 +34,20 @@
   </select>
 </span>
 
+<span>
```
Comment (mtrojan-ub) [inline themes/tuefind/templates/RecordTab/itemfulltextsearch.phtml:37]: Please also add a `style="display: none;"` attribute to the span, just to be sure that the span itself does not have any side effects on the layout.

File: themes/tuefind/templates/RecordTab/itemfulltextsearch.phtml [hunk 5]
```diff
@@ -60,4 +50,24 @@
 </form>
 <span id="fulltext_snippets">
   <div id="snippet_place_holder_<?=$this->driver->getUniqueID()?>"></div>
-</span>
\ No newline at end of file
+</span>
+
```
Comment (LysogorAndGmail) [inline themes/tuefind/templates/RecordTab/itemfulltextsearch.phtml:54]: $('#ItemFulltextSearchForm').submit(function(){}) does not work in tuefind.js

**Pull request merged**